crates/symbolicator/src/services/symbolication.rs

```diff
@@ -1942,9 +1904,38 @@ impl SymbolicationActor {
             let result = loop {
                 iterations += 1;
 
-                let result = self
-                    .stackwalk_minidump_with_cfi(minidump.clone(), &cfi_caches)
-                    .await?;
+                let result = match self
+                    .stackwalk_minidump_with_cfi(&minidump_file, &cfi_caches)
+                    .await
+                {
+                    Ok(res) => res,
+                    Err(err) => {
+                        if let Some(dir) = self.diagnostics_cache.cache_dir() {
+                            if let Some(file_name) = minidump_file.file_name() {
+                                let path = dir.join(file_name);
+                                match minidump_file.persist(&path) {
```

Comment:
> Note: Temporary files cannot be persisted across filesystems.

Are we sure this is never going to be the case?  It often is the case that `/tmp` is a different filesystem.  And afaik our cache directory is a persistent volume so very likely to be a different filesystem to wherever tempfile defaults.

/cc @beezz: can you confirm whether this would work or not?

Reply:
I changed this to save the files in `$CACHE/tmp` which is also used for other tmp files already, so that won’t run into this particular problem.